code/game/objects/items/emags.dm

```diff
@@ -73,6 +94,25 @@
 			return FALSE
 	return TRUE
 
+/// What happens if a crewmember uses this card
+/obj/item/card/emag/proc/crew_action(mob/user)
+	if(!HAS_TRAIT(user, TRAIT_EMAG_USER))
+		crew_uses += 1
+	if(!prob(crew_uses * 25))
+		return
+	to_chat(user, span_warning("You hear a buzzing from [src] 'Non-authorised personel detected. Terminating card'."))
+	user.audible_message(span_boldwarning("[src] beeps ominously!"))
+	playsound(src, 'sound/items/timer.ogg', 35, vary = FALSE)
+	// Think fast chucklenuts
+	is_exploding = TRUE
+	addtimer(CALLBACK(src, PROC_REF(detonate), user), 2 SECONDS)
+
+/// What happens when this card is exploded! Used if you want to add any unique explosion effects to the card.
+/obj/item/card/emag/proc/detonate(mob/user)
+	user.visible_message(span_boldwarning("[src]'s battery ruptures!"))
+	explosion(src, light_impact_range = 1, explosion_cause = src)
```

Comment:
If you really, REALLY, want to dissuade people using this as crew AND as an antag checker, and don't want to fuck with the probabilities, give this a dev range so it gibs the user. If sec start using this to antag check and get a normal crew member gibbed as a result, they're going to be in massive shit. Sure, it is a feels bad for the sap who got the emag, but it also gives our failed antagonist a final comeuppance against the crew.